drivers/sensor/qdec_nrfx/qdec_nrfx.c

```diff
@@ -195,8 +195,8 @@ static int qdec_nrfx_init(struct device *dev)
 
 	LOG_DBG("");
 
-	IRQ_CONNECT(DT_NORDIC_NRF_QDEC_QDEC_0_IRQ,
-		    DT_NORDIC_NRF_QDEC_QDEC_0_IRQ_PRIORITY,
+	IRQ_CONNECT(DT_NORDIC_NRF_QDEC_QDEC_0_IRQ_0,
```

Comment:
Why this does not need the _INST_ infix, @rlubos ?
Also, it looks there is no board in the tree that enables the qdec device (unless I am missing something)

Reply:
> Why this does not need the INST infix, @rlubos ?

Not all symbols w/o *INST* infix were deprecated. I've discussed this with @anangl, he recommended this approach.

> Also, it looks there is no board in the tree that enables the qdec device (unless I am missing something)

I suppose that's why it wasn't spotted earlier - we do have such board in nrfconnect.

Reply:
Yes, I have seen the board in nrf-connect :) Thanks